CMakeLists.txt

```diff
@@ -236,6 +236,7 @@ else()
     endif()
     message(STATUS "Adding ${VIRTUALENV_HOME_DIR} to CMAKE_PREFIX_PATH")
     list(APPEND CMAKE_PREFIX_PATH ${VIRTUALENV_HOME_DIR})
+    find_package(Rocisa 0.1.0 EXACT REQUIRED HIP LLVM PATHS "${INSTALLED_TENSILE_PATH}")
```

Comment:
I'm having trouble figuring out what this means. Specifically, I'm not sure why the "HIP LLVM" part is here and I suspect that is being ignored by CMake.

Reply:
Sorry, this is copied from the Tensile find_package below. Will remove it, thanks!

Reply:
It can be a bit fiddly to use find_package for a fixed PATH that should not be found anywhere else.

Recommend something like this: `find_package(Rocisa 0.1.0 EXACT REQUIRED CONFIG NO_DEFAULT_PATH PATHS "${INSTALLED_TENSILE_PATH}")`

(without `CONFIG`, cmake uses a more elaborate legacy procedure, and in both cases, iirc some system level things are checked before the PATHS you provide unless if you specify NO_DEFAULT_PATH)

Reply:
Fixed